engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -608,13 +609,16 @@ protected void serviceStart() throws Exception
 
     try {
       Configuration config = getConfig();
+      YarnConfiguration conf = new YarnConfiguration();
```

Comment:
Configuration is needed by WebApp server. We could probably do something like `YarnConfiguration conf = StramClientUtils.getYarnConfiguration(config);`
or
`YarnConfiguration conf = new YarnConfiguration(config);`
However in both cases there is not much significant difference unless & until config is instanceof YarnConfiguration.